src/build.rs

```diff
@@ -30,6 +30,7 @@ fn main() {
     let out_dir = PathBuf::from(env::var_os("OUT_DIR").expect("outdir"));
     let config_dir = out_dir.join("include");
     let vendor = root.join("vendor");
+    println!("cargo:rerun-if-changed={}", vendor.to_str().unwrap());
```

Comment:
Please use .display() on paths

Reply:
Cargo doesn't support non-utf8 paths in case you were wondering, but I can change it.